assets/js/theme/global/quick-search.js

```diff
@@ -62,7 +62,7 @@ export default function () {
                 }, 100);
             }
         });
-    }, 200);
+    }, 1200);
```

Comment:
Settling in on 1200 ms as a middle ground between the average typing speed, slowness while typing on touchscreens and a still acceptable wait time before fetching results.

Reply:
as a suggestion: can we use a variable for this number which will explain it or just add a comment so it will make sense for future devs

Reply:
Thanks @bc-alexsaiannyi. Added the change.

Reply:
Can this be approved and merged ? I don't see any need for testing on this.

Reply:
@saurabhgedam its merged